.gitignore

```diff
@@ -7,3 +7,6 @@
 /doc/
 /vendor/gems/
 /Gemfile.local
+.ruby-gemset
```

Comment:
It looks like you might've inadvertently updated the gitignore in this PR? Probably to match your own workflow?

Reply:
Removed